docs/topics/native/native-objc-interop.md

```diff
@@ -136,67 +147,70 @@ package my.library
 
-You can specify a more idiomatic name in Swift or Objective-C, instead of renaming the Kotlin declaration.
-Use the `@ObjCName` annotation that instructs the Kotlin compiler to use a custom Objective-C and Swift name for methods
-or parameters.
+| Kotlin       | Swift          | Objective-C   |
+|--------------|----------------|---------------|
+| `equals()`   | `isEquals(_:)` | `isEquals:`   |
+| `hashCode()` | `hash`         | `hash`        |
+| `toString()` | `description`  | `description` |
 
-> Using this annotation requires [opt-in](opt-in-requirements.md).
->
-{type="note"}
+[See an example with data classes in the Kotlin-Swift interopedia](https://github.com/kotlin-hands-on/kotlin-swift-interopedia/blob/main/docs/classesandinterfaces/Data%20classes.md).
+
+You can specify a more idiomatic name in Swift or Objective-C, instead of renaming the Kotlin declaration with
+the [`@ObjCName` annotation](#custom-declaration-names).
 
 ### Errors and exceptions
 
-Kotlin has no concept of checked exceptions, all Kotlin exceptions are unchecked.
-Swift has only checked errors. So if Swift or Objective-C code calls a Kotlin method
-which throws an exception to be handled, then the Kotlin method should be marked
-with a `@Throws` annotation specifying a list of "expected" exception classes.
+Kotlin has no concept of checked exceptions, all Kotlin exceptions are unchecked. Swift has only checked errors.
+So if Swift or Objective-C code calls a Kotlin method which throws an exception to be handled, the Kotlin method
+should be marked with the `@Throws` annotation specifying a list of "expected" exception classes.
+
+When compiling to the Objective-C/Swift framework, non-`suspend` functions that have or inherit the `@Throws` annotation
+are represented as `NSError*`-producing methods in Objective-C and as `throws` methods in Swift.
+Representations for `suspend` functions always have`NSError*`/`Error` parameter in completion handler.
 
-When compiling to the Objective-C/Swift framework, non-`suspend` functions that have or inherit the
-`@Throws` annotation are represented as `NSError*`-producing methods in Objective-C
-and as `throws` methods in Swift.  Representations for `suspend` functions always have
-`NSError*`/`Error` parameter in completion handler.
+When Kotlin function called from Swift/Objective-C code throws an exception which is an instance of one of
+the `@Throws`-specified classes or their subclasses, it is propagated as `NSError`.
+Other Kotlin exceptions reaching Swift/Objective-C are considered unhandled and cause program termination.
 
-When Kotlin function called from Swift/Objective-C code throws an exception
-which is an instance of one of the `@Throws`-specified classes or their subclasses,
-it is propagated as `NSError`. Other Kotlin exceptions reaching Swift/Objective-C
-are considered unhandled and cause program termination.
+`suspend` functions without `@Throws` propagate only `CancellationException` as `NSError`.
```

Comment:
```suggestion
`suspend` functions without `@Throws` propagate only `CancellationException` (as `NSError`).
```

Without punctuation, this might read, “and other exceptions are propagated as something else.”